test/src/unit-capi-array.cc

```diff
@@ -699,6 +700,387 @@ TEST_CASE_METHOD(
   }
 }
 
+TEST_CASE_METHOD(
+    ArrayFx,
+    "C API: Test array with encryption, no with_key API",
+    "[capi], [array], [encryption], [without_key]") {
```

Comment:
Although we do this in many places, let's use `"[capi][array][encryption][without_key]"` (i.e., no commas) moving forward.